server/container/cli/src/main/java/org/apache/james/cli/probe/impl/JmxSieveProbe.java

```diff
@@ -75,8 +78,11 @@ public void removeSieveQuota(String user) throws Exception {
     }
 
     @Override
-    public void addActiveSieveScript(String user, String name, String script) throws Exception {
-        sieveRepositoryManagement.addActiveSieveScript(user, name, script);
+    public void addActiveSieveScript(String user, String toFileName, String fromScriptFileName) throws Exception {
+        if (!toFileName.endsWith(".siv") || !toFileName.endsWith(".sieve"))
+            toFileName = toFileName + ".sieve";
```

Comment:
Please avoid variable reallocation.... You can do it by introducing a **String sanitizeScriptName(String fileName)** method.